src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -150,14 +150,24 @@ public async Task InitAsync()
                 {
                     Email = await _stateService.GetRememberedEmailAsync();
                 }
-                var deviceIdentifier = await _appIdService.GetAppIdAsync();
-                IsKnownDevice = await _apiService.GetKnownDeviceAsync(Email, deviceIdentifier);
                 CanRemoveAccount = await _stateService.GetActiveUserEmailAsync() != Email;
-                await _deviceActionService.HideLoadingAsync();
+                IsKnownDevice = await _apiService.GetKnownDeviceAsync(Email, await _appIdService.GetAppIdAsync());
             }
             catch (Exception ex)
             {
-                HandleException(ex);
+                if(ex is ApiException apiException
+                    && apiException.Error.StatusCode == System.Net.HttpStatusCode.Unauthorized)
+                {
+                    _logger.Exception(ex);
```

Comment:
The background here is that we modified the `/knowndevice` endpoint to pass the email address in a header instead of the request.  Rather than make it backward compatible and keep that privacy risk out there, we are going to handle the error that occurs on the client when hitting an old server.  This should only happen when hitting an old server from a new device, which would be a self-hosted scenario or a server rollback.  These shouldn't be cases that would happen on a regular basis, which is I suspect why the logs are there - to still track whether we are getting 401s unexpectedly.

Reply:
Perfect then 😄